app/build.gradle

```diff
@@ -32,6 +32,13 @@ dependencies {
     implementation 'androidx.recyclerview:recyclerview:1.1.0'
     implementation 'de.hdodenhof:circleimageview:3.1.0'
     implementation "androidx.cardview:cardview:1.0.0"
+    implementation 'androidx.recyclerview:recyclerview:1.1.0'
+    implementation 'androidx.cardview:cardview:1.0.0'
+    implementation 'com.google.android.material:material:1.0.0'
+    def nav_version = "2.3.0-alpha04"
+    implementation "androidx.navigation:navigation-fragment:$nav_version"
+    implementation "androidx.navigation:navigation-ui:$nav_version"
+    implementation 'com.github.xiangtailiang:tablayout:1.0.0'
```

Comment:
this is a very old library and you don't use it in your project and this good 
we have to use libraries that are trusted and valid for our case 
so please remove this